client/components/eventList.jsx

```diff
@@ -16,13 +16,18 @@ export default class EventList extends React.Component {
   }
 
   componentDidMount() {
-    fetch('/api/events')
-      .then(res => res.json())
+    fetch('/api/events', { redirect: 'follow' })
```

Comment:
I don't think this `init` argument is needed as the `redirect` mode defaults to `'follow'`, no?